### ISSUE TITLE
Alt text is not descriptive [../predictions files]

### ISSUE THREAD

**Issue by timngyn** [aws-amplify/docs#7068]
**Describe the content issue**:
Images listed below do not have descriptive/informative alt text. For markdown the alt text is located between the square brackets, `![alt text](...)`.

The list below isn't exhaustive since I found these images based on the word count for the alt text, but it should be a good starting point to make our alt text better for people who use screen readers.

Some resources to help write alt text:
- https://webaim.org/techniques/alttext/
- https://www.a11yproject.com/posts/are-you-making-these-five-mistakes-when-writing-alt-text/

&nbsp;

https://github.com/aws-amplify/docs/blob/main/src/fragments/lib-v1/predictions/js/identify-text.mdx:
```
  119  
  120: ![Image](/images/IdentifyTable.png)
  121  
```

&nbsp;

https://github.com/aws-amplify/docs/blob/main/src/fragments/lib/predictions/js/identify-text.mdx:
```
  127  
  128: ![Image](/images/IdentifyTable.png)
  129  
```
